.github/workflows/actions.yaml

```diff
@@ -1,11 +1,11 @@
 name: Run Tests
-on: 
+on:
   - pull_request
 jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.17.x, 1.18.x]
+        go-version: [1.19.x, 1.23.x]
```

Comment:
Fly-by change: the minimum supported version in `go.mod` is already 1.19, and I've introduced a use of `any` that causes this library to fail to compile on 1.17.

Reply:
I think there may be references to 1.17 and 1.18 elsewhere, breaking CI 🙈

Reply:
Hmm, no. Not sure why CI is blocked on actions that no longer exist. 🤔